### ISSUE TITLE
False positive of RCS1207 (Use method group instead of lambda).

### ISSUE THREAD

**Issue by JosefPihrt** [JosefPihrt/Roslynator#257]
```csharp
public class Foo
{
    public string Bar()
    {
        Foo foo = null;

        Func<string> func = () => foo.Bar();

        return null;
    }
}
```
